### PR TITLE
Add more tests and update data file

### PR DESCRIPTION
**Pull request by pombredanne**
This PR adds tests from the Mozilla test suite for get_tld(strict=True/False) and get_sld()
@hiratara @KnitCode your review is muched welcomed before I push a release!

Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>

### CODE REVIEW

**Comment by pombredanne** [comment]
Note: Python 2.7 tests are failing for now https://travis-ci.org/nexB/python-publicsuffix2/jobs/627296065?utm_medium=notification&utm_source=github_status

**Comment by KnitCode** [comment]
> Note: Python 2.7 tests are failing for now https://travis-ci.org/nexB/python-publicsuffix2/jobs/627296065?utm_medium=notification&utm_source=github_status

i think they should pass if the unicode versions are all explicitly converted... i can test later when i get home... but i think replacing 

    def test_get_tld_IDN_labels7(self):
        assert '中国' == publicsuffix.get_tld('www.食狮.中国')

with 

    def test_get_tld_IDN_labels7(self):
        assert u'中国' == publicsuffix.get_tld(u'www.食狮.中国')

should get all of them to work.... 

otherwise the rest looks good to me. more tests, more better. thanks for the author notation.

File: tests_mozilla.py [hunk 1]
```diff
@@ -0,0 +1,811 @@
...
+        assert 'k12.ak.us' == publicsuffix.get_tld('k12.ak.us')
+
+    def test_get_tld_US_K128(self):
+        assert 'k12.ak.us' == publicsuffix.get_tld('test.k12.ak.us')
+
+    def test_get_tld_US_K129(self):
+        assert 'k12.ak.us' == publicsuffix.get_tld('www.test.k12.ak.us')
+
+    def test_get_tld_IDN_labels1(self):
+        assert 'com.cn' == publicsuffix.get_tld('食狮.com.cn')
```
Comment (KnitCode) [inline tests_mozilla.py:238]: each of the unicode strings has to be replaced with unicode string (wrong term, i'm sure) and the unit tests will pass... 
'食狮.com.cn' --> u'食狮.com.cn'

I made the changes locally and all the unit tests passed with that change.

**Review by KnitCode — CHANGES_REQUESTED** [review]
each of the unicode strings has to be replaced with unicode string (wrong term, i'm sure) and the unit tests will pass...
'食狮.com.cn' --> u'食狮.com.cn'

**Comment by pombredanne** [comment]
@KnitCode Thank you for your review!
I moved to using proper unicode throughout on all Python version in the tests and also made a few other cosmetic fixes and we are all green now.

**Review by KnitCode — APPROVED** [review]
looks good to me!

**Comment by pombredanne** [comment]
All merged, tagged and pushed as https://pypi.org/project/publicsuffix2/2.20191219/
@hiratara @KnitCode Thank you ++

**Comment by hiratara** [comment]
Thank you for adding tests. It looks good to me.

It seems a bit weird that no test case produces the same result as test_psl.txt. However, it is natural because we have no function that provides the same effect as `checkPublicSuffix`.

**Pull request merged**